aea/test_tools/network.py

```diff
@@ -48,3 +48,9 @@ def get_host() -> str:
     finally:
         s.close()
     return IP
+
+
+def is_port_in_use(port: int) -> bool:
```

Comment:
A simple test would be nice

Reply:
it is used in tests here https://github.com/valory-xyz/open-aea/blob/8acf11140157870dc292fa14be90b00c2ecefc81/tests/test_test_tools/test_network.py#L33-L38

It does not cover the positive case though